proposal/more-math-functions.md

```diff
@@ -76,17 +78,73 @@ length of the `n`-dimensional vector that has components equal to each of the
 inputs. Since the inputs' units may all be different, the output takes the unit
 of the first input.
 
+## Semantics
+
+### Built-in Module Variables
+
+Variables defined in built-in modules are not modifiable. As such, this proposal
+modifies the semantics of [Executing a Variable Declaration][] within the
+[Variables spec][] to read as follows:
+
+[Executing a Variable Declaration]: ../spec/variables.md#executing-a-variable-declaration
+[Variables spec]: ../spec/variables.md
+
+To execute a `VariableDeclaration` `declaration`:
+
+* Let `value` be the result of evaluating `declaration`'s `Expression`.
+
+* Let `name` be `declaration`'s `Variable`.
+
+* **Let `resolved` be the result of [resolving a variable][] named `name`.**
+
+[resolving a variable]: ../spec/modules.md#resolving-a-member
+
+* If `name` is a `NamespacedVariable` and `declaration` has a `!global` flag,
+  throw an error.
+
+* **Otherwise, if `resolved` is a variable from a built-in module, throw an
+  error.**
+
+* Otherwise, if `declaration` is outside of any block of statements, *or*
+  `declaration` has a `!global` flag, *or* `name` is a `NamespacedVariable`:
+
+  * ~~Let `resolved` be the result of [resolving a variable][] named `name` using
+    `file`, `uses`, and `import`.~~
+
+  [...]
```

Comment:
This gets detected as a potential link reference, which is causing the tests to fail. Maybe write it `(...)` instead.

Reply:
Done.